addon/mixins/dom.js

```diff
@@ -133,29 +133,33 @@ export default Mixin.create({
     }
 
     // We cannot use Array.findIndex as we cannot rely on babel/polyfill being present
-    for (let i = 0; i < this._listeners.length; i++) {
-      let listener = this._listeners[i];
+    for (let i = 0; i < this._listeners.length; i += 5) {
```

Comment:
I'd even add a constant to represent the magic 5, something like `LISTENER_INFO_SIZE`. This will help hint to people that the indexes represent a grouping of information, and the array is not merely a contiguous set of disjointed items.